src/libopensc/card-dtrust.c

```diff
@@ -70,6 +70,27 @@ static struct dtrust_supported_ec_curves {
 };
 // clang-format on
 
+/* copied from iso7816.c */
```

Comment:
This is already copied in cardos and gids driver. Can we make it public function in some common file to avoid the need to copy it to one more place? It makes in unmaintainable.

Reply:
This was my initial thought, but I didn't want to throw it in question as other drivers already copied that code. But now I mad this function public and removed all duplicate code.